.gitignore

```diff
@@ -1 +1,2 @@
 /.idea/
+.DS_Store
```

Comment:
I'm doubtful, is that .DS_Store or .DS_STORE? And why wouldn't it get ignored by default anyway?

Reply:
i don't know, it didnt and its annoying to remove it every time :D Its, `.DS_Store`